java/ql/src/semmle/code/java/FlowSources.qll

```diff
@@ -0,0 +1,281 @@
+/**
```

Comment:
This feels seems to be a duplicate of the existing `FlowSources.qll` file.

Reply:
I don’t think so. The ```FlowSources.qll``` file is missing for SpringMVC for remote input. The ```SpringServletInputParameterSource``` class can only get the remote part of the user's input,  for example the following code is in the ```source``` Won't get it.
```java
@GetMapping(value = "index")
public void index(String request){
     System.out.println(request);
}
```
I added the ```SpringMVC``` class to the ```FlowSources.qll``` file, mainly to verify whether the class annotation where the method parameters are obtained uses the ``Mapping``` and sub-annotations.
You can try the above code with the following statement.
```java
import java
import semmle.code.java.dataflow.FlowSources

from RemoteFlowSource rfs
select rfs
```